cmake/modules/FindTaglib.cmake

```diff
@@ -11,7 +11,7 @@
 # For details see the accompanying COPYING-CMAKE-SCRIPTS file.
 
 if(NOT TAGLIB_MIN_VERSION)
-    set(TAGLIB_MIN_VERSION "1.7")
+    set(TAGLIB_MIN_VERSION "1.7.0")
```

Comment:
Can you explain why this is necessary for your PR?

Reply:
trying to find out why the shell build does not work. it has version 1.11.1 installed but thinks 1.7 is newer. other ideas ?

Reply:
works local

Reply:
ahh, the broken locale was the culpit